metrics_utils/src/lib.rs

```diff
@@ -230,6 +242,319 @@ impl BackfillerMetricsConfig {
+            db_account_get_buckets_latency: mk_histogram(),
+            db_account_get_latests_latency: mk_histogram(),
+
+            bbgm_consistency_grpc_client: Arc::new(
+                BubblegumConsistencyGrpcClientMetricsConfig::new(),
+            ),
+            acc_consistency_grpc_client: Arc::new(AccoountConsistencyGrpcClientMetricsConfig::new()),
+
+            found_missing_bubblegums: Gauge::default(),
+            found_missing_accounts: Gauge::default(),
+        }
+    }
+
+    pub fn register(&self, registry: &mut Registry) {
+        self.bbgm_consistency_grpc_client.register(registry);
+        self.acc_consistency_grpc_client.register(registry);
+        registry.register(
+            "db_bubblegum_get_grand_epochs_latency",
+            "Time to query bubblegum grand epochs",
+            self.db_bubblegum_get_grand_epochs_latency.clone(),
+        );
+        registry.register(
+            "db_bubblegum_get_epochs_latency",
+            "Time to query bubblegum epochs",
+            self.db_bubblegum_get_epochs_latency.clone(),
+        );
+        registry.register(
+            "db_bubblegum_get_changes_latency",
+            "Time to query bubblegum changes",
+            self.db_bubblegum_get_changes_latency.clone(),
+        );
+        registry.register(
+            "db_account_get_grand_buckets_latency",
+            "Time to query account NFT grand buckets",
+            self.db_account_get_grand_buckets_latency.clone(),
+        );
+        registry.register(
+            "db_account_get_buckets_latency",
+            "Time to query account NFT buckets",
+            self.db_account_get_buckets_latency.clone(),
+        );
+        registry.register(
+            "db_account_get_latests_latency",
+            "Time to query account NFT last states",
+            self.db_account_get_latests_latency.clone(),
+        );
+
+        registry.register(
+            "found_missing_bubblegums",
+            "Number of identified missing bubblegum changes",
+            self.found_missing_bubblegums.clone(),
+        );
+        registry.register(
+            "found_missing_accounts",
+            "Number of identified missing account NFT last states",
+            self.found_missing_accounts.clone(),
+        );
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct BubblegumConsistencyGrpcClientMetricsConfig {
+    pub peers_bubblegum_get_grand_epochs_for_tree_latency: Histogram,
+    pub peers_bubblegum_get_grand_epochs_latency: Histogram,
+    pub peers_bubblegum_get_epochs_latency: Histogram,
+    pub peers_bubblegum_get_changes_latency: Histogram,
+
+    pub peers_bubblegum_get_grand_epochs_for_tree_errors: Family<MetricLabel, Counter>,
+    pub peers_bubblegum_get_grand_epochs_errors: Family<MetricLabel, Counter>,
+    pub peers_bubblegum_get_epochs_errors: Family<MetricLabel, Counter>,
+    pub peers_bubblegum_get_changes_errors: Family<MetricLabel, Counter>,
```

Comment:
it looks more like an added label to me. Peers_sync_latency(protocol: "bubblegum/account", method/endpoint: "get_grand_epochs/get_epochs/get_changes"